xbmc/guilib/GraphicContext.cpp

```diff
@@ -983,16 +984,17 @@ void CGraphicContext::SetMediaDir(const std::string &strMediaDir)
   m_strMediaDir = strMediaDir;
 }
 
+
+void CGraphicContext::SetStereoMode(RENDER_STEREO_MODE mode)
+{
+  m_stereoMode = mode;
+  g_application.m_pPlayer->TriggerUpdateResolution();
```

Comment:
So stereoscopic manager gets a call on playback start, playback stop and on user actions (e.g. GUI OSD). What should it do from those?
Can it call g_graphicContext.SetVideoResolution?
Can it call g_application.m_pPlayer->TriggerUpdateResolution ?